src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1816,7 +1816,27 @@
         GeneratedJavaTokenTypes.LITERAL_synchronized;
 
     /**
-     * The {@code volatile} keyword.
+     * The {@code volatile} keyword. This may be used as a
+     * modifier of a field.
+     *
+     * <p>For example:</p>
+     *
+     * <pre>
+     *   private volatile int variable;
```

Comment:
preceding spaces can be removed

and lets use other variable name, `variable` and `VARIABLE_DEF` can confuse users, i would suggest to use simple one or two-letter names

Reply:
Done, Thank you for the reviews.